store/mem.go

```diff
@@ -0,0 +1,136 @@
+package store
+
+import (
+	"bytes"
+	"encoding/json"
+
+	"github.com/graphql-go/graphql"
+
+	bolt "go.etcd.io/bbolt"
+)
+
+var dbPath = "default.db"
+
+const (
+	schemaBucket      = "schema"
+	schemaGraphBucket = "schemaGraph"
+	triggerBucket     = "trigger"
+)
+
+func saveElement(tenantID string, bucketID string, element []byte) error {
+	db, err := bolt.Open(dbPath, 0666, nil)
+	if err != nil {
+		return err
+	}
+	defer db.Close()
+
+	updateErr := db.Update(func(tx *bolt.Tx) error {
+		bucket, err := tx.CreateBucketIfNotExists([]byte(bucketID))
+		if err != nil {
+			return err
+		}
+		err = bucket.Put([]byte(tenantID), element)
+		if err != nil {
+			return err
+		}
+		return nil
+	})
+	if updateErr != nil {
+		return updateErr
+	}
+	return nil
+}
+
+func fetchElement(tenantID string, bucketID string) ([]byte, error) {
+	db, err := bolt.Open(dbPath, 0666, nil)
+	if err != nil {
+		return nil, err
```

Comment:
Should there be some context added to the err message? `fmt.Errorf("failed to open bolt db: %w", err)` ?

Reply:
Seems like just `err` is being returned everywhere. Perhaps it's sufficient